{{cookiecutter.project_dashed}}-src/ansible/group_vars/all

```diff
@@ -1,16 +1,18 @@
-
 use_pvt_dns: false
 
 project_ident: {{ cookiecutter.project_namespace }}
+project_ident_env: {{ cookiecutter.project_namespace }}-{{ '{{' }} app_environment {{'}}'}}
 
-app_db_host: "{{ cookiecutter.database_pvt_host }}"
 {% raw %}
+app_db_host: "{{ lookup('dig', '+short', 'sierra.pvt.level12.net', '@ns1.linode.com')}}"
+
 # Host and ip setup
 network_domain: level12.net
-host_fqdn: "{{inventory_hostname}}.{{network_domain}}"
-host_ip: "{{ lookup('pipe', 'dig +short ' + host_fqdn) }}"
-host_fqdn_pvt: "{{inventory_hostname}}.pvt.{{network_domain}}"
-host_ip_pvt: "{{ lookup('pipe', 'dig +short ' + host_fqdn_pvt) }}"
+server_dns_name: "{{ inventory_hostname.split('-')|first }}"
```

Comment:
This seems fragile.  There is no reason a host name couldn't be something like 'foo-bar'.  At least leave a comment about what assumptions you are making that cause you to do this.